packages/amplication-client/src/Project/ArchitectureConsole/hooks/useModelOrganizer.tsx

```diff
@@ -449,37 +449,23 @@ const useModelOrganization = ({ projectId, onMessage }: Props) => {
 
   const searchPhraseChanged = useCallback(
     (searchPhrase: string) => {
-      if (searchPhrase === "") {
-        nodes.forEach((x) => (x.hidden = false));
-        edges.forEach((e) => (e.hidden = false));
-      } else {
-        const searchModelGroupNodes = nodes.filter(
-          (node) =>
-            node.type === "modelGroup" &&
-            !node.data.payload.name.includes(searchPhrase) &&
-            node.id !== currentEditableResourceNode?.id
-        );
-
-        searchModelGroupNodes.forEach((x) => {
-          x.hidden = true;
-          const childrenNodes = nodes.filter(
-            (node: EntityNode) => node.parentNode === x.id
-          );
-
-          childrenNodes.forEach((x) => (x.hidden = true));
+      nodes.forEach((x) => (x.data.highlight = false));
 
-          const nodeEdges = edges.filter((e) => {
-            return childrenNodes.find((n) => e.source === n.id);
-          });
+      if (searchPhrase !== "") {
+        const searchModelGroupNodes = nodes.filter((node) =>
```

Comment:
these are not just modelGroupNodes
```suggestion
        const searchResults = nodes.filter((node) =>
```